mongodbatlas/projects.go

```diff
@@ -233,3 +234,24 @@ func (s *ProjectsServiceOp) AddTeamsToProject(ctx context.Context, projectID str
 
 	return root, resp, err
 }
+
+// RemoveUserFromProject removes user from a project
+// See more: https://docs.atlas.mongodb.com/reference/api/project-remove-user/
```

Comment:
[nit] please add a line between them or it will render poorly on the godoc (optional to fix this file if they are all like that)
```suggestion
// RemoveUserFromProject removes a user from a project
//
// See more: https://docs.atlas.mongodb.com/reference/api/project-remove-user/
```